test/e2e/simple_test.go

```diff
@@ -354,9 +354,9 @@ func TestPodAffinity(t *testing.T) {
 					// will wait for pod to scheduled until timeout.
 					RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
 						NodeSelectorTerms: []corev1.NodeSelectorTerm{
-							corev1.NodeSelectorTerm{
+							{
```

Comment:
To remain consistent with above changes, could you consider updating this to below format

```
NodeSelectorTerms: []corev1.NodeSelectorTerm{{
  MatchExpressions: []corev1.NodeSelectorRequirement{{
    Key: .....
  }},
}}
...
```

Reply:
I **really** wish `gofmt -s` did this.  It is going to be a pain to edit the sockpuppet PR, so I'd rather merge the pure tool-based change and follow up with the more aggressive reduction.